src/test/java/org/json/junit/JSONParserConfigurationTest.java

```diff
@@ -208,6 +218,15 @@ public void verifyMaxDepthThenDuplicateKey() {
      */
     private List<String> getNonCompliantJSONList() {
         return Arrays.asList(
+            "[]asdf",
+            "[]]",
+            "[]}",
+            "[][",
+            "[]{",
+            "[],",
+            "[]:",
+            "[],[",
+            "[],{",
```

Comment:
Please add this entry to the noncompliant list:
`"[[a]]"`
It will break the unit tests since JSONTokener.nextValue() does not forward the jsonParserConfiguration object when creating a new JSONArray, so some additional work will be needed.

Reply:
Will do.